paasta_tools/docker_wrapper.py

```diff
@@ -336,6 +433,8 @@ def main(argv=None):
 
     if env_args.get("PIN_TO_NUMA_NODE"):
         argv = append_cpuset_args(argv, env_args)
+    else:
+        argv = cap_memory_arg(argv)
```

Comment:
if we do go this route: this doesn't seem like the right place to add this

iirc: this wrapper is used by the vanishingly small amount of tasks that we still run on mesos + we don't really use the `PIN_TO_NUMA_NODE` feature, so this memory cap would be applied to everything still on mesos/using this wrapper